docs/v2/manual.md

```diff
@@ -1424,6 +1434,166 @@ In this example the flag could be used like this :
 
 Side note: quotes may be necessary around the date depending on your layout (if you have spaces for instance)
 
+### Migration Guide: v1 to v2
+
+v2 has a number of breaking changes but converting is relatively
+straightforward: make the changes documented below then resolve any
+compiler errors.  We hope this will be sufficient for most typical
+users.
+
+If you find any issues not covered by this document, please post a
+comment on [Issue 921](https://github.com/urfave/cli/issues/921) or
+consider sending a PR to help improve this guide.
+
+#### Flags before args
+
+In v2 flags must come before args.  This is more POSIX-compliant.  You
+may need to update scripts, user documentation, etc.
+
+This will work:
+
+```
+cli hello --shout rick
+```
+
+This will not:
+
+```
+cli hello rick --shout
+```
+
+#### Import string changed
+
+* OLD: `import "github.com/urfave/cli"`
+* NEW: `import "github.com/urfave/cli/v2"`
+
+Check each file for this and make the change.
+
+Shell command to find them all: `fgrep -rl github.com/urfave/cli *`
+
+#### Flag aliases are done differently.
+
+Change `Name: "foo, f` to "Name: "foo", Aliases: []string{"f"}`
```

Comment:
```suggestion
Change `Name: "foo, f` to `"Name: "foo", Aliases: []string{"f"}`
```

Reply:
I think this is even better:

    Change `Name: "foo, f"` to `Name: "foo", Aliases: []string{"f"}`